notes/notesModel.js

```diff
@@ -0,0 +1,35 @@
+const db = require('../data/dbConfig.js');
+
+module.exports = {
+    get: function(id) {
+        if(id) {
+            return db('notes').where('id',id)
```

Comment:
I'm pretty sure you can just do `.first()` here instead of the `.then` statement below

Reply:
Worked great.  Thank you